### DISCUSSION TITLE
Roles - allow one input on create but do not show on edit/view

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3132] [Archive]
Is it possible to have such roles configured so that I allow 1 field on creating but not allow to view/edit that field?

Currently, if I disable that field on `Read` and `Update` I don't see it on `Create`.

**Comment by benhaynes** [comment]
You can set the permissions to be enabled for a field on the create action, but I don't think we have separate READ vs WRITE permissions per field any more. @rijkvanzanten Thoughts on adding that back?

![Screen Shot 2020-11-19 at 10 04 22 AM](https://user-images.githubusercontent.com/522079/99683693-a04abb80-2a4e-11eb-8eba-03632890b7f2.png)

**Comment by botzill** [comment]
Is there a way to achieve this via custom web hook?

**Reply by benhaynes** [reply]
Well, many things are possible with Webhooks, you could enable the field in Permissions, and then check if a value was submitted in a webhook and throw an error if they try to update it. That said, we'd want to update the core codebase to allow for this without webhooks.

**Comment by botzill** [comment]
I see, yes that I can do. But I want to hide the field completely not even
allow to input data there. It's a password field.

**Reply by benhaynes** [reply]
Then you can turn the field off completely for `create` in permissions. Also, please keep all responses inline within Discussions. 😄

**Reply by botzill** [reply]
The idea is that I want to have it on Create but not on Edit/View
